scrapers/Staxus.yml

```diff
@@ -0,0 +1,382 @@
+              "Salvadoran": "El Salvador"
+              "Sammarinese": "San Marino"
+              "Samoan": "Samoa"
+              "Saudi Arabian": "Saudi Arabia"
+              "Saudi": "Saudi Arabia"
+              "Scottish": "Scotland"
+              "Senegalese": "Senegal"
+              "Serbian": "Serbia"
+              "Seychellois": "Seychelles"
+              "Sierra Leonean": "Sierra Leone"
+              "Singapore": "Singapore"
+              "Singaporean": "Singapore"
+              "Slovak": "Slovakia"
+              "Slovene": "Slovenia"
+              "Slovenian": "Slovenia"
+              "Solomon Island": "Solomon Islands"
+              "Somali": "Somalia"
+              "Somalilander": "Somaliland"
+              "South African": "South Africa"
+              "South Georgia Island": "South Georgia and the South Sandwich Islands"
+              "South Ossetian": "South Ossetia"
+              "South Sandwich Island": "South Georgia and the South Sandwich Islands"
+              "South Sudanese": "South Sudan"
+              "Spanish": "Spain"
+              "Sri Lankan": "Sri Lanka"
+              "Sudanese": "Sudan"
+              "Surinamese": "Suriname"
+              "Svalbard resident": "Svalbard"
+              "Swati": "Eswatini"
+              "Swazi": "Eswatini"
+              "Swedish": "Sweden"
+              "Swiss": "Switzerland"
+              "Syrian": "Syrian Arab Republic"
+              "Taiwanese": "Taiwan"
+              "Tajikistani": "Tajikistan"
+              "Tanzanian": "Tanzania"
+              "Thai": "Thailand"
+              "Timorese": "Timor-Leste"
+              "Tobagonian": "Trinidad and Tobago"
+              "Togolese": "Togo"
+              "Tokelauan": "Tokelau"
+              "Tongan": "Tonga"
+              "Trinidadian": "Trinidad and Tobago"
+              "Tunisian": "Tunisia"
+              "Turkish": "Turkey"
+              "Turkmen": "Turkmenistan"
+              "Turks and Caicos Island": "Turks and Caicos Islands"
+              "Tuvaluan": "Tuvalu"
+              "Ugandan": "Uganda"
+              "Ukrainian": "Ukraine"
+              "Uruguayan": "Uruguay"
+              "Uzbek": "Uzbekistan"
+              "Uzbekistani": "Uzbekistan"
+              "Vanuatuan": "Vanuatu"
+              "Vatican": "Vatican City State"
+              "Venezuelan": "Venezuela"
+              "Vietnamese": "Vietnam"
+              "Wallis and Futuna": "Wallis and Futuna"
+              "Wallisian": "Wallis and Futuna"
+              "Welsh": "Wales"
+              "Yemeni": "Yemen"
+              "Zambian": "Zambia"
+              "Zimbabwean": "Zimbabwe"
+              "Åland Island": "Åland Islands"
+      Image: 
+        selector: //div[@class="item active"]/img/@src
+        postProcess:
+          - replace:
+              - regex: ^
+                with: "https:"
+      Details:
+        selector: //div[@class="video-descr__section col-md-12"]//p[2]/text()
+
```

Comment:
```yaml
      Birthdate:
        selector: //p[@class="flex-item"]/b[text()="Date of birth:"]/following-sibling::span[1]/text()
        postProcess:
          - replace:
              - regex: '(\d+)\S+\s+([a-zA-Z]+)\D+(\d+)'
                with: "$1 $2 $3"
          - parseDate: 2 January 2006
 ```
Birthdate can also be added
The above seems to work in most cases, not when the date is partial though (found cases with missing the day part)
Should we ignore it in that case or use `1` as the day?